en/test_and_ci/continous_integration.md

```diff
@@ -1,3 +1,212 @@
+| [PX4_misc/Firmware_update_submodules](http://ci.px4.io/job/PX4_misc/job/Firmware_update_submodules/) | Create PRs updating the submodules to latest | Manual | In Jenkins |
+| [PX4/*](http://ci.px4.io/job/PX4/) | Main CI pipelines for PX4 ecosystem | NA | NA |
+| [PX4/PX4-Autopilot](http://ci.px4.io/job/PX4/job/PX4-Autopilot/) (On Github shown as **continuous-integration/jenkins/branch or pr-head**) | Updates the `master` metadata files in the S3 bucket | Push to `main` | [Jenkinsfile](https://github.com/PX4/PX4-Autopilot/blob/main/Jenkinsfile) |
+| [PX4/PX4-user_guide](http://ci.px4.io/job/PX4/job/PX4-user_guide/) | Builds the Html version of the user guide from the markdown format and deploys to the web | Updates in `main` (latest) or `v1.*` (past releases) branches | [PX4-user_guide/Jenkinsfile](https://github.com/PX4/PX4-user_guide/blob/main/Jenkinsfile) |
+
+For the full view of all different pipelines, visit [ci.px4.io/](http://ci.px4.io/).
+
+### Hardware testing Jenkins
+
+A Webhook is configured to [px4-jenkins.dagar.ca](http://px4-jenkins.dagar.ca:8080/) for every event of Pushes (new commits, branches) and Pull Requests in PX4-Autopilot repository.
+
+| Pipeline Name | Description | Trigger | Script |
+|---|---|---|---|
+| [PX4-Autopilot](http://px4-jenkins.dagar.ca:8080/job/PX4-Autopilot/) (On Github shown as **continuous-integration/jenkins/branch**) | Runs hardware test script | Pushes & Pull Requests (all Webhook events) | [.ci/Jenkinsfile-hardware](https://github.com/PX4/PX4-Autopilot/blob/main/.ci/Jenkinsfile-hardware) |
+
+Hardware rack is composed of actual hardware connected via USB, each connected with separate Jenkins nodes currently as following:
+
+- `cubepilot_cubeorange_test`: Cubepilot CubeOrange
+- `cuav_x7pro_test`: CUAV X7 Pro
+- `px4_fmu-v4_test`: FMU v4
+- `px4_fmu-v4pro_test`: FMU v4 Pro
+- `px4_fmu-v5_debug`: FMU v5 with debug flag, prints out debug information
+- `px4_fmu-v5_stackcheck`: FMU v5 compiled with stackcheck, detects stack overflow, etc.
+- `px4_fmu-v5_test`: FMU v4
+- `nxp_fmuk66-v3_test`: FMU k66 v3
+
+Note that `_test` label for the target means ... (TODO)
+
+It performs the following tests on each build node targets:
+
+- Build: Build bootloader and firmware binary files
+- Flash: Flash the hardware via USB, JLink, and other methods (specific to hardware)
+- Tests: Various tests regarding sensors, commander module, uorb topics, etc.
+- Status: Reboot and check filesystem (/proc, /dev, etc), module, system commands, and do quick IMU calibration
+- Print topics: Print out selected set of uORB topic data (for debugging purposes)
+
+This CI is crucial for detecting failures in hardware that can't be detected via Software CI tools, such as hardfault, NuttShell, NuttX bugs, etc.
+
+For the full test script, visit [.ci/Jenkinsfile-hardware](https://github.com/PX4/PX4-Autopilot/blob/main/.ci/Jenkinsfile-hardware).
```

Comment:
Added general information about the hardware rack here. Anything else you want me to add? @mnumanuyar